### ISSUE TITLE
Allow PublishSymbols to publish binaries (dll, exe) and well as pdbs

### ISSUE THREAD

**Issue by davidjward30** [Microsoft/vsts-tasks#809]
We usually publish binaries (dll, exe) to our symbol server in addition to pdb files. The reason being is that it's much easier to debug crash dumps using only the symbol sever.

The script failed trying to index sources on a .dll.

**Comment by chrisrpatterson** [comment]
This is not something I have ever seen.  Could be give me some additional information on how you use this today?

**Comment by davidjward30** [comment]
Before TFS, our build system used symstore.exe to publish binaries and pdbs to a symbol server. The advantage of having the binaries on the symbol server is that given just a crash dump (e.g one automatically uploaded to our crash dump server by a customer), we can open in up in Visual Studio or WinDbg and get the callstack without the additional step of locating the original build.

When we migrated to TFS2010 build, we modified the workflow to include binaries as well.

So naturally as we look to migrate to TFS2015 build we want to do the same thing.
